.evergreen/config.yml

```diff
@@ -2177,6 +2178,19 @@ buildvariants:
     # Test standalone and sharded only on 4.4.
     - '.4.4'
 
+- matrix_name: "tests-pyopenssl-pypy"
+  matrix_spec:
+    platform: debian92
+    python-version: ["pypy", "pypy3.5", "pypy3.6"]
+    auth: "auth"
+    ssl: "ssl"
+    pyopenssl: "*"
+  display_name: "PyOpenSSL ${platform} ${python-version} ${auth}"
+  tasks:
+    - '.replica_set !.2.6 !.3.0 !.3.2 !.3.4'
+    # Test standalone and sharded only on 4.4.
```

Comment:
Wouldn't this also test RS?

Reply:
Can you also explain why we want to only run this on RS for 3.6, 4.0, 4.2 ?

Reply:
This config tests:
- 3.6+ with replica sets
- 4.4 with standalone, repl, sharded.

We don't test standalone+sharded on other server versions because it would be overkill.